src/transformers/training_args.py

```diff
@@ -641,6 +655,10 @@ class TrainingArguments:
     label_smoothing_factor: float = field(
         default=0.0, metadata={"help": "The label smoothing epsilon to apply (zero means no label smoothing)."}
     )
+    optim: OptimizerNames = field(
+        default=OptimizerNames.ADAMW_HF.value,
```

Comment:
@sgugger, what would you like here? Everywhere else with enum we set the default to the actual string, so here it can be:
- `OptimizerNames.ADAMW_HF.value`
- `adamw_hf`

Use the latter for consistency?

Reply:
I personally prefer the string values, yes.

Reply:
@stas00 @sgugger I changed the default to the string value.